utility/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
   buildToolsVersion "29.0.2"
 
   defaultConfig {
-    minSdkVersion 19
+    minSdkVersion 21
```

Comment:
This doesn't seem ideal--we're still technically supporting KitKat (at least for builds). Is this actually necessary?

Reply:
reverted to 19.